modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/file/AbstractFileIO.java

```diff
@@ -62,9 +64,19 @@ else if (n == 0 || i > 0) {
 
                     if (time == 0)
                         time = System.nanoTime();
-                    else if ((System.nanoTime() - time) >= U.millisToNanos(MAX_IO_TIMEOUT_MS))
-                        throw new IOException(write && (position + i) == size() ? "Failed to extend file." :
-                            "Probably disk is too busy, please check your device.");
+                    else if ((System.nanoTime() - time) >= U.millisToNanos(MAX_IO_TIMEOUT_MS)) {
+                        String errMsg = " operation unsuccessful, write timeout exceeds the maximum IO timeout ("
```

Comment:
You use "write timeout" even in case of read operation.

Reply:
Corrected.